sound/soc/intel/boards/sof_sdw.c

```diff
@@ -351,6 +351,14 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.init = sof_sdw_rt711_init,
 		.exit = sof_sdw_rt711_exit,
 	},
+	{
```

Comment:
it looks like we are moving to a finer granularity, this is no longer a codec_info_list but codec_endpoint_list.